karma.shared.js

```diff
@@ -2,16 +2,19 @@
 
 /* This is the shared karma config */
 
+var merge = require('lodash.merge');
 var RewirePlugin = require('rewire-webpack');
-var webpackConfig = require('./webpack.config.js');
+var webpackConfig = require('./webpack.config');
 
+
+var newWebpackConfig = merge({}, webpackConfig);
```

Comment:
Thinking more about this, I'd prefer `object-assign` just so we don't add a new dep (even though it's a dev dep).